SwissKnife/gcloud/GCloudStorage.py

```diff
@@ -13,14 +13,50 @@ def __init__(self, logger: logging.Logger = logging.getLogger("GCloudStorage")):
         which contains the path of the SA that will be used for authentication.
         It's also a must to have an active Internet connection, otherwise the
         connection with the gcloud servers can not be performed.
+        Also, this object needs the enviroment variable "BUCKET_PATH". It will extract the
+        BUCKET_NAME and the BUCKET_PATH_PREFIX using the "info" module. If BUCKET_PATH is not
+        defined, an exception will be raised.
 
         :param logger: Logger object to use, defaults to logging.getLogger("GCloudStorage")
         :type logger: logging.Logger, optional
         """
+
+        if BUCKET_NAME is None:
+            raise RuntimeError("BUCKET_PATH environment variable not available.")
+
         self.logger = logger
         self.logger.info(f"Building a new gcloud Storage client for bucket {BUCKET_NAME}")
         self.storage_client = gcloud.Client()
         self.bucket = self.storage_client.get_bucket(BUCKET_NAME)
+    
+    def download_to_file(self, gs_path:str , local_path: str, use_bucket_path_prefix:bool=True, is_binary:bool=False):
```

Comment:
I miss documentation here. I also find a little bit confusing the function name: it is the same as the original Google Storage function, maybe it could be `download_to_local_file`?

Reply:
Ups, you are right. I forgot it. And yes, that name will be better.